### DISCUSSION TITLE
Daily 01/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#510] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by rpgrca** [comment]
Ayer terminé con casi todas las tareas, encontré algunos errores que no permitían modificar los artículos y cosas así que ya arreglé. Hoy tendría que armar bien el script de sql y arreglar lo del total del carrito si no está hecho. También estoy actualizando un poco los README. Hoy por ahí hago la tarea de inglés para mañana y a las 20 hacemos la reunión del sprint review.

**Comment by NataliaAlvarezIspc** [comment]
Ayer termine con las tareas que tenía de formularios, también hice la tarea que tenía de Inglés y por la noche tuve clases.
Hoy voy  estar revisando la documentación y testeando el sitio.

**Comment by BraianTroncoso** [comment]
Hoy hice un poco de testing de cosas de la web sumado que intento ponerme el dia con unas cuestiones de python. También tuve clases de programación con el ISPC y participé del sprint review.
